sentry/src/routes/event_aggregate.rs

```diff
@@ -0,0 +1,47 @@
+use chrono::{serde::ts_milliseconds_option, DateTime, Utc};
+use hyper::{Body, Request, Response};
+use serde::Deserialize;
+
+use primitives::{adapter::Adapter, sentry::EventAggregateResponse, Channel};
+
+use crate::{db::list_event_aggregates, success_response, Application, ResponseError, Session};
+
+#[derive(Deserialize)]
+pub struct EventAggregatesQuery {
+    #[serde(default, with = "ts_milliseconds_option")]
+    after: Option<DateTime<Utc>>,
+}
+
+pub async fn list_channel_event_aggregates<A: Adapter>(
+    req: Request<Body>,
+    app: &Application<A>,
+) -> Result<Response<Body>, ResponseError> {
+    let channel = req
+        .extensions()
+        .get::<Channel>()
+        .expect("Request should have Channel");
```

Comment:
Should it crash here or return an error?

Reply:
Crash. As we don't have a direct way to pass a loaded channel from the `load_channel` middleware :(